src/Compilers/Core/Portable/Compilation/SemanticModel.cs

```diff
@@ -86,6 +86,11 @@ public IOperation GetOperation(SyntaxNode node, CancellationToken cancellationTo
 
         protected abstract IOperation GetOperationCore(SyntaxNode node, CancellationToken cancellationToken);
 
+        public ImmutableArray<Operations.BasicBlock> GetControlFlowGraph(Operations.IBlockOperation body)
```

Comment:
Let's add PROTOTYPE marker to document this too (since public API). Thanks

Reply:
Added a comment and opened [https://github.com/dotnet/roslyn/issues/24310](https://github.com/dotnet/roslyn/issues/24310)

---
In reply to: [162143405](https://github.com/dotnet/roslyn/pull/24263#discussion_r162143405) [](ancestors = 162143405)